pkg/swagger/types.go

```diff
@@ -245,7 +245,16 @@ type Schema struct {
 	ExternalDocs         *ExternalDocumentation `json:"externalDocs,omitempty"`
 	Example              interface{}            `json:"example,omitempty"`
 
-	ClientFlatten bool `json:"x-ms-client-flatten,omitempty"`
+	ClientFlatten bool              `json:"x-ms-client-flatten,omitempty"`
+	XMSEnum       *XMSEnumExtension `json:"x-ms-enum,omitempty"`
```

Comment:
nit: Enum *Enum `json:"x-ms-enum,omitempty"`

Reply:
Enum already exists. `XMSEnum` is a separate field.